src/pages/task/metadata/index.tsx

```diff
@@ -277,6 +278,12 @@ export const Metadata: React.VFC<Props> = ({
           ))}
         </span>
       ) : null}
+      {resetWhenFinished && (
+        <P2 data-cy="reset-when-finished-container">
```

Comment:
nit. the `depends-on-container` is only named that way because it can contain multiple dependent tasks

```suggestion
        <P2 data-cy="reset-when-finished">
```